lib/mnesia/doc/guides/mnesia_chap4.md

```diff
@@ -843,6 +847,11 @@ or less results than specified with `NObjects` can be returned in the result
 list, even the empty list can be returned even if there are more results to
 collect.
 
+There is also [`select_reverse/1,2,3,4`](`mnesia:select_reverse/2`), should you
+want to traverse from the end of the result set. This is only applicable to
+`ram_copies` or `disc_copies` tables of type `ordered_set`. For other table
+configurations, it behaves the same as `mnesia:select/1,2,3,4`.
```

Comment:
I think you can add link here to mnesia:select/1,2,3,4 (same as above for mnesia:select_reverse/1,2,3,4).